build_runner/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   args: ">=1.4.0 <2.0.0"
   async: ">=1.13.3 <3.0.0"
-  build: ">=1.0.0 <1.1.0"
+  build: ">=1.1.0 <1.0.0"
```

Comment:
<1.2.0

Reply:
Done. Also moved minimum back to 1.0.0 since I didn't have to change any Dart code here.